app/models.py

```diff
@@ -94,6 +94,7 @@ class Price(Base):
     id = Column(Integer, primary_key=True, index=True)
 
     product_code = Column(String, nullable=True, index=True)
+    product_name = Column(String, nullable=True, index=True)
```

Comment:
Do we really want to index it? The database can grow quickly, and it's a string field

Reply:
you're right, fixed here : https://github.com/openfoodfacts/open-prices/pull/83/commits/4b6f20956806fe34f2ff4b52aea0849c93ab1510